cmd/swctl/app/cmd_manage.go

```diff
@@ -370,11 +373,31 @@ func runManageCmd(cli Cli, opts ManageOptions, args []string) error {
 		finalConf = mainConf
 	}
 
-	// format final output
 	if opts.Format == "" {
 		opts.Format = "yaml"
 	}
-	if err := formatAsTemplate(cli.Out(), opts.Format, finalConf); err != nil {
+
+	var fileName string
+	var writer io.Writer
+
+	if opts.OutputFile == "" {
+		fileName = fmt.Sprintf("%s.%s", entityName, opts.Format)
+	} else {
+		outputFilePrefix := strings.Split(opts.OutputFile, ".")
+		fileName = fmt.Sprintf("%s.%s", outputFilePrefix[0], opts.Format)
```

Comment:
This will not work when outputfile will contain more than one dot. When i think about it, you can also put absolute path to file end it should work. In that case is quite possible that some directory has dot inside (i.e. /home/john/.config/stonework/config.yaml). Please take the prefix until the last dot in the given outputfile string.